src/components/Cart.js

```diff
@@ -1,36 +1,39 @@
 import React, { useEffect, useState } from "react";
-
 import axios from "axios";
 import "./Cart.css"
-import { useNavigate } from "react-router-dom";
-
 
-const Cart = ({currentUser, isLoggedIn, token}) => {
+const Cart = () => {
     const [cart, setCart] = useState([]);
-    const navigate = useNavigate()
 
     const fetchOrderProduct = async () => {
-        const orderProduct = await axios.get("/api/orders/cart", 
-            {
-                headers: { Authorization: `Bearer ${localStorage.getItem('token')}` }
-            }
+        const orderProduct = await axios.get(
+            "/api/orders/cart", 
+            { headers: { Authorization: `Bearer ${localStorage.getItem('token')}` }}
         )
         setCart(orderProduct.data)
         console.log(orderProduct.data)
     };
     useEffect(fetchOrderProduct, []);
 
+    const checkout = async () => {
```

Comment:
This is all that I really had to add, to call our create-checkout-session endpoint, which returns a URL that we navigate to for Stripe checkout.

Reply:
Noted how the code works and this call. Awesome work! Will want to try using this all again in other projects.